docs/how-to-install.md

```diff
@@ -6,145 +6,170 @@ license: "This software is licensed under the Apache License version 2."
+To configure the Gravitino server, you can update this file.
+For a full list of the Gravitino server configurations, see [Gravitino Server Configurations](./gravitino-server-config.md).
 
-The Gravitino server environment configuration file is `conf/gravitino-env.sh`. Gravitino exposes several environment variables. You can modify them in this file.
+### Configure the Gravitino server log
 
-#### Configure Apache Gravitino catalogs
+Gravitino uses [Log4j2](https://logging.apache.org/log4j/2.x/) as the logging system.
+The `conf/log4j2.properties` file provides the Gravitino server log configurations.
+To configure the Gravitino server log, you can update this file.
 
-Gravitino supports multiple catalogs. You can configure the catalog-level configurations by modifying the related configuration file in the `catalogs/<catalog-provider>/conf` directory. The configurations you set here apply to all the catalogs of the same type you create.
+### Configure the Gravitino server environment variables
 
-For example, if you want to configure the Hive catalog, you can modify the file `catalogs/hive/conf/hive.conf`. The detailed configurations are listed in the specific catalog documentation.
+Gravitino exposes several environment variables through the `conf/gravitino-env.sh` file.
+To configure these environment variables, you can update this file.
+
+### Configure the Gravitino catalogs
+
+Gravitino supports multiple catalogs.
+You can configure the catalog-level configurations by updating the related configuration file in the `catalogs/<catalog-provider>/conf` directory.
+For example, the `catalogs/hive/conf/hive.conf` file provides configurations for the Hive catalog.
+The configurations you set in the catalog configuration file will apply to all the catalogs of the same type that you created.
+For detailed configurations about each catalog, see related catalog documentation.
 
 :::note
-Gravitino takes the catalog configurations in the following order:
+Gravitino takes the catalog configurations in the following order of precedence:
 
-1. Catalog `properties` specified in catalog creation API or REST API.
+1. Catalog `properties` specified in the catalog creation API or REST API.
 2. Catalog configurations specified in the catalog configuration file.
 
-The catalog `properties` can override the catalog configurations specified in the configurationfile.
+The catalog `properties` overrides the catalog configurations specified in the configuration file.
 :::
 
-Gravitino supports passing in catalog-specific configurations if you add `gravitino.bypass.`. For example, if you want to pass in the HMS-specific configuration `hive.metastore.client.capability.check` to the underlying Hive client in the Hive catalog, add the `gravitino.bypass.` prefix.
+Gravitino supports passing in catalog-specific configurations by adding the `gravitino.bypass.` prefix to the specific catalog configurations.
```

Comment:
configurations -> configuration

Reply:
why use singular from? do you mean  Gravitino supports passing in only one configuration?

Reply:
See above. Configuration is generally the plural of configuration, and it is uncommon to use "configurations".